graph/graph.go

```diff
@@ -41,6 +43,11 @@ type Edge struct {
 	Metadata map[string]interface{} // app-specific data
 }
 
+type NamespaceInfo struct {
+	models.Namespace
```

Comment:
Do we really need models.Namespace or could it just be the name, as before?  I don't think anything but the name is used outside of Options processing.

Reply:
Hmmm... Yes, it can be only the name. I'll change.